### PR TITLE
Devtest

### PR DESCRIPTION
**Pull request by Rohit-PX**
**What type of PR is this?**
integration-test

**What this PR does / why we need it**:
Provide a way to run integration tests by replacing stork, stork_test images 

**Does this PR change a user-facing CRD or CLI?**:
no

**Is a release note needed?**:
no

**Does this change need to be cherry-picked to a release branch?**:
no

### CODE REVIEW

File: specs/stork-deployment.yaml [hunk 1]
```diff
@@ -110,7 +110,7 @@ spec:
         # automatically update schedulerName
         #- --app-initializer=true
         imagePullPolicy: Always
-        image: openstorage/stork:2.2.4
+        image: DEV_STORK_IMAGE
```
Comment (disrani-px) [inline specs/stork-deployment.yaml]: Let's not add placeholders to the specs, someone should be able to apply these specs directly

File: test/integration_test/test-deploy.sh [hunk 2]
```diff
@@ -217,6 +217,10 @@ fi
 kubectl delete -f /testspecs/stork-test-pod.yaml
 kubectl create -f /testspecs/stork-test-pod.yaml
 
+
+echo "ROHIT==> Test spec\n"
```
Comment (disrani-px) [inline test/integration_test/test-deploy.sh]: Remove debug

Comment (Rohit-PX) [reply test/integration_test/test-deploy.sh]: Fixed.

File: specs/stork-deployment.yaml (hunk 1 above)
Comment (Rohit-PX) [reply specs/stork-deployment.yaml]: Removed placeholders.

**Pull request merged**